_drafts/2017-06-07-checkers.md

```diff
@@ -0,0 +1,115 @@
+---
+name: checkers
+layout: post_discourse
+title: Tackling the Research Compendium at #runconf17
+authors:
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+  - name: Alice Daish
+    url: http://twitter.com/alice_data
+  - name: Laura DeCicco
+    url: http://twitter.com/DeCiccoDonk 
+  - name: Nistara Randhawa
+    url: http://twitter.com/nistara_r
+  - name: Jennifer Thompson
+    url: http://twitter.com/jent103
+  - name: Nick Tierney
+    url: https://twitter.com/nj_tierney
+categories:
+  - blog
+tags:
+  - unconf
+  - testing
+  - review
+  - packages
+  - software
+  - community
+  - meetings
+---
+
+Two years ago at [\#runconf15], there was a great discussion about best practices for organizing R-based analysis projects that yielded a [nice guidance document] describing *research compendia*. Compendia, as we described them, were minimal products of reproducible research, using parts of R package structure to organize the inputs, analyses, and outputs of research projects.
+ 
+Since then, we've seen more examples and models of research compendia emerge (the organization of such projects is [something of an obsession] for some of the community). In parallel, there's been much progress on a number of fronts with R *packages*: ROpenSci's [package review process] has expanded and we've worked out many kinks. Infrastructure for automated testing of package code has been developed and field tested. So at [\#runconf17](unconf17.ropensci.org), we wanted to see how much of this progress in review, testing, and automation could apply to research compendia.
+ 
+It turns out there's a *lot* to do here, and a lot of interest! We put up a proposal as a [GitHub issue]; before the unconf even began, the thread had over 50 posts and the discussion had yielded two design documents led by Hadley Wickham on [research compendium organization] and [automated build systems]. There were easily four or five projects wrapped up in the proposal.
+ 
+The thread also revealed many schools of thought. As [Nick] put it, "The problem with [opinionated analysis development] is that everyone has an opinion." We never reached consensus on basic issues like directory structure and build systems.
+
+![](http://imgur.com/RpjyfL8.jpeg)
```

Comment:
- add photo caption (this is the only one without)
- add photographer credit to each photo (Nistara?). I'm not sure best way to do this. I added alt-tags in my unconf post but they don't show up so I need to fix mine too